### DISCUSSION TITLE
Ellipsis文本省略的expandText值换成图标后，内容没有撑到列宽就自动收缩了，为什么呢？

### DISCUSSION THREAD

**Discussion by githubqh** [ant-design/ant-design-mobile#5300] [Q&A] [answered]
如果expandText值是字符串就没有问题，换成图标就会复现
![image](https://user-images.githubusercontent.com/105548468/172747701-9eff2654-4949-44a8-994f-aab758b26156.png)
https://codesandbox.io/s/admiring-nova-4dg21u?file=/src/App.tsx
import React from "react";
import { Ellipsis } from "antd-mobile";
import { DownOutline } from "antd-mobile-icons";
![image](https://user-images.githubusercontent.com/105548468/172748265-3f789cfb-665d-4aff-90dc-3faa4a70b89e.png)

const content =
  "蚂蚁的企业级产品是一个庞大且复杂的体系。这类产品不仅量级巨大且功能复杂，而且变动和并发频繁，常常需要设计与开发能够快速的做出响应。同时这类产品中有存在很多类似的页面以及组件，可以通过抽象得到一些稳定且高复用性的内容。";

export default () => {
  return (
    <>
      <Ellipsis
        direction="end"
        content={content}
        expandText={<DownOutline />}
        collapseText="收起"
      />
    </>
  );
};

**Comment by awmleer** [comment, marked as answer]
`expandText` 只支持 `string` 类型，请读文档，而且你的编辑器都给你标红线提示你了

**Reply by githubqh** [reply]
> `expandText` 只支持 `string` 类型，请读文档，而且你的编辑器都给你标红线提示你了

后续能否支持图标呢，文字不好看，还有expandText的文字是否能支持调节大小，不要和正文字体大小一样

**Reply by awmleer** [reply]
因为底层实现的原因，大概率不会支持的
